content/workload-reduction-toolkit/address-workload-issues/feedback-and-marking/secondary-school-feedback-policy.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Secondary school feedback policy
+layout: /markdown_resource.html.erb
+---
+
+# Secondary school feedback policy
+
+<strong class="govuk-tag">Example</strong>
+
+{inset-text}
+
+## School details
+
+**School name:** St Joseph’s RC High School
+
+**Location:** Bolton
+
+**Phase:** Secondary
+
+**Number of pupils:** 840
+
+**Contact details:** Email the school office at <office@st-josephs.bolton.sch.uk>
+
+{/inset-text}
+
+<div class="govuk-grid-row dfe-width-container">
+  <div class="govuk-grid-column-full">
+    <div class="info-box">
+      <div class="info-box__corner">
+        <img src="/assets/images/bullseye.svg" alt="Bullseye icon">
+      </div>
+      <h2 class="govuk-heading-m">
+        Impact and outcomes
+      </h2>
+      <p>
+        This policy reduced the time teachers spent marking where it was not having an impact on pupil progress. 
+      </p>
+      <p>
+        Marking for progress has encouraged some departments to amend their assessment schemes at key stage 3.
+      </p>
+      <p>
+        For pupils, it allowed misconceptions to be highlighted and gave them time to understand what they need to do to progress. 
+      </p>
+    </div>
+  </div>
+</div>
+
+## Background from Tony McCabe, Headteacher
+
+{inset-text}
+We developed this policy because we found that teachers spent a lot of time marking, but not all of this was having an impact on pupil progress. We decided to focus on marking where the input from the teacher was proportionate to the impact on the progress. 
+
+We shared these time saving examples across the school and out of this ‘best practice’ grew our impact marking policy, which focuses solely on marking for progress. 
+{/inset-text}
+
+## Our feedback principles  
+
+All feedback: 
+
+* informs pupil progress
+  
+* has a positive impact on pupil outcomes
+
+* is a good use of teachers’ time 
+
+The more immediate the feedback the greater the impact. All subject areas have identified a bare minimum standard in terms of regularity of feedback, taking into consideration the frequency of lessons, and the nature of the subject.  
+
+## When to give written and verbal feedback 
+
+Other than a tick for completion, a teacher should only write in books if:
+
+* they are pointing out a good attribute that a pupil must continue to use
+  
+* they are pointing out a literacy error
+  
+* they are pointing out an area for development (this can be highlighted in the text)
+  
+* the work will conclude with a GCSE style mark (where appropriate) and a tangible next step to move learning on 
+
+Where verbal feedback is given teachers should only record the feedback if it will reinforce the implementation of the next step, not purely for the benefit of book looks. 
```

Comment:
This sentence is quite long:
'Where verbal feedback is given, teachers should only record it if it will reinforce the implementation of the next step. It should not be provided for the benefit of book looks.' (Is 'book looks' a commonly used phrase in teaching? If so, it should be in single quotation marks)

Reply:
it is. fixed with marks